### ISSUE TITLE
Using CAMITAX without an internet connection

### ISSUE THREAD

**Issue by lovettse** [abremges/CAMITAX#21]
Is there any way to install/run CAMITAX on a machine with no connection to the internet (data can be transferred via external hard drive)?

**Comment by abremges** [comment]
Theoretically yes. Can you run `nextflow run CAMI-challenge/CAMITAX/init.nf --db /path/to/db/folder` on some other machine with access to the internet? Then all you have to do is to copy `/path/to/db/folder` to the machine without internet access (and either install all dependencies manually or somehow copy the Docker containers – which can be a hassle).

Please let us know if this solution worked for you. Please note that you'll lose the ability to quickly update the software and reference data if you go down this road.

**Comment by lovettse** [comment]
I finally got to spend some time trying to work on this. We know the problem with keeping things updated on the unconnected systems well, but it is unavoidable.

I ran the init.nf command and brought the dbs over. I have the requirements installed in a conda environment. I tried running `nextflow run CAMITAX-master/main.nf --db /path/to/db --input genomes --x fna`. I get:  

`WARN: The config file defines settings for an unknown process: cache_.*|taxonomy|summary`  
`Input genomes:`  
`[warm up] executor > local`  

If the problem is my lack of experience with Nextflow, I can work on that, but do I at least have the command syntax right?

Thank you
